docs/core/porting/upgrade-assistant-uwp-framework.md

```diff
@@ -0,0 +1,524 @@
+
+The [.NET Upgrade Assistant](upgrade-assistant-overview.md) is a command-line tool that can assist with upgrading .NET Framework WPF apps to .NET 6. This article provides:
+
+- What to expect from the tool?
+- Things to know before starting your migration journey 
+- A demonstration of how to run the tool against a UWP app
+- Troubleshooting tips
+
+For more information on how to install the tool, see [Overview of the .NET Upgrade Assistant](upgrade-assistant-overview.md).
+
+## What to expect from the tool?
+
+We’re working on an update to the .NET Upgrade Assistant which is an open-source tool to help you automate migration of .NET Native UWP apps to .NET 6, including updating from WinUI 2 to WinUI 3. This is a command line tool that leverages .NET Upgrade assistant and will live as an extension in the same github repo [here](https://github.com/dotnet/upgrade-assistant.git). This tool aims to provide a simple migration experience by 
+
+- Backing up your UWP project 
+- Convert project file (csproj) to SDK style
+- Clean up NuGet package references 
+- Update the target framework and NuGet packages 
+- Update package.appxmanifest
+- Removal of properties and items that are no longer required
+- Add template files such as App.Xaml, MainWindow.Xaml and publish profiles with options to build packaged and unpackaged app
+- Make namespace changes and add MainPage navigation
+- An attempt to detect and fix apis that have changed from UWP to Windows App SDK
+- An attempt to detect and mark apis that are either no longer supported or cannot be automatically converted with TODO comments and CLI messages
+
+We aim to provide migration guidance in form of warning messages within the tool and TODO comments within your project as the tool tries to migrate the project. In this way, you’ll always be in control of your migration. And for the APIs where complete automation is not possible, plan is to add //TODO comments for the developers to know where the work will be needed. A typical //TODO comment will also include a link to our existing migration documentation. Please check the Task list within the Visual Studio to see all the action items as TODO comments.
+
+## Things to know before starting your migration journey
+
+This tool currently supports C# and in most cases the app will require additional effort to complete the migration. The goal of the tool will be to convert your project and code so that it can compile, but features that require additional developer investigation will have TODO comments. Please check out the what’s supported page [here] (https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/what-is-supported)] before you begin your migration journey.
+
+Additionally, you may choose to wait for the next version of .Net UA tool before you start migrating your app because of the current limitations of the tool:
```

Comment:
Fix .NET and don't reference the tool as UA
```suggestion
Additionally, you may choose to wait for the next version of .NET Upgrade Assistant tool before you start migrating your app, because of the current limitations of the tool:
```